File: javascript01/step02/test01.js
/* 주제: 연산자 - 산술 연산자
- +, -, *, /, %, **
*/
"use static"

var a = 7;
var b = 2;

console.log(a + b); // 9
console.log(a - b); // 5
console.log(a * b); // 14
console.log(a / b); // 3.5
console.log(a % b); // 1
console.log(b ** 3); // 거듭제곱
                    // node v7이상 지원. 아직 크로스브라우징 안됨

console.log(10 / 0); // Infinity(무한)
console.log(-10 / 0); // -Infinity

File: java01/src/step17/Test01.java
/* 예외처리 : 예외 처리를 하지 않는 경우 */
package step17;

public class Test01 {
  public static float divide(float a, float b) {
    return a / b;
  }

  public static void main(String[] args) {
    System.out.println(divide(10f, 2f)); // 5.0
    System.out.println(divide(0, 2));    // 0.0
    System.out.println(divide(10, 0));   // Infinity 
  }

}
